Models/removeComment.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.removeComment = (commentID) => {
+  return db
+    .query(`DELETE from comments WHERE comment_id = $1`, [commentID])
+    .then((result) => {
+      if (!result.rows) {
```

Comment:
it doesn't look like your query returns anything, so will your result.rows always be falsy?

Reply:
how do you know if the comment has been deleted?